package.json

```diff
@@ -106,5 +107,10 @@
     "tsx": "^3.11.0",
     "typescript": "^4.8.4",
     "vitest": "^0.24.3"
+  },
+  "overrides": {
+    "graphql-voyager": {
+      "graphql": "^15.8.0"
```

Comment:
why is this needed?

Reply:
GraphQL-Voyager needs graphql to be `16.8.0` and above by default. We are overwriting the same here so that the users don't have to provide `--force` flag while installing the same.